project-base/config/routes/shopsys_front.yml

```diff
@@ -22,6 +22,19 @@ front_cart_delete:
     requirements:
         cartItemId: \d+
 
+front_cart_box_detail:
+    path: /cart/box-detail
+    defaults:
+        _controller: App\Controller\Front\CartController:boxDetailAction
```

Comment:
Shouldn't this route be only for ajax too? Same way as `front_cart_delete_ajax`

Reply:
Yes, it should, thx